modules/sql/src/main/resources/db/migration/V033__Magnitude_Band.sql

```diff
@@ -0,0 +1,59 @@
+--
+-- Name: magnitude_system; Type: TYPE; Schema: public; Owner: postgres
+--
+
+-- Note: this may need to be turned into a table once we start taking into
+-- account the units needed for ITC Watts and Ergs.
+CREATE TYPE magnitude_system AS ENUM (
+    'Vega',
+    'AB',
+    'Jy'
+);
+
+ALTER TYPE magnitude_system OWNER TO postgres;
+
+COMMENT ON TYPE magnitude_system IS 'Integrated brightness units';
+
+--
+-- Name: e_magnitude_band; Type: TABLE; Schema: public; Owner: postgres
+--
+
+CREATE TABLE e_magnitude_band (
+    id             identifier            PRIMARY KEY,
+    short_name     character varying(2)  NOT NULL,
+    long_name      character varying(14) NOT NULL,
+    center         integer               NOT NULL,
+    width          integer               NOT NULL,
+    default_system magnitude_system      NOT NULL
+);
+
+ALTER TABLE e_magnitude_band OWNER TO postgres;
+
+COMMENT ON COLUMN e_magnitude_band.center IS 'nm';
+COMMENT ON COLUMN e_magnitude_band.width  IS 'nm';
+
+--
+-- Data for Name: e_magnitude_band; Type: TABLE DATA; Schema: public; Owner: postgres
+--
+
+COPY e_magnitude_band (id, short_name, long_name, center, width, default_system) FROM stdin;
+SloanU	u	UV	356	46	AB
```

Comment:
I hadn't realized that you could just put the values as plain text here. neat

Reply:
I didn't know that either. Cool.